config.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     port: process.env.REDIS_PORT || '6379',
     host: process.env.REDIS_HOST || '127.0.0.1'
   },
+  dataService: {
+    url: process.env.DATA_SERVICE_URL || 'http://localhost:8080',
```

Comment:
Kubernetes automatically exposes some environment variables courtesy of the `data-service` `svc`; it might be good to use them if we can:

```
nodejs@app-86f496d654-g587r:/app$ env | grep DATA_SERVICE
DATA_SERVICE_PORT_10443_TCP_ADDR=10.10.24.115
DATA_SERVICE_SERVICE_PORT_PORT_1=10080
DATA_SERVICE_SERVICE_PORT_PORT_2=10443
DATA_SERVICE_PORT_10443_TCP_PORT=10443
DATA_SERVICE_PORT_10443_TCP_PROTO=tcp
DATA_SERVICE_SERVICE_HOST=10.10.24.115
DATA_SERVICE_PORT_10080_TCP=tcp://10.10.24.115:10080
DATA_SERVICE_SERVICE_PORT=10080
DATA_SERVICE_PORT=tcp://10.10.24.115:10080
DATA_SERVICE_PORT_10080_TCP_PROTO=tcp
DATA_SERVICE_PORT_10443_TCP=tcp://10.10.24.115:10443
DATA_SERVICE_PORT_10080_TCP_PORT=10080
DATA_SERVICE_PORT_10080_TCP_ADDR=10.10.24.115
```